__tests__/Port.test.js

```diff
@@ -0,0 +1,39 @@
+const Port = require("../src/Port");
+
+describe("Port", () => {
+  describe("ports and ships", () => {
+    let port;
+    let ship;
+    let titanic;
+    let queenMary;
+
+    beforeEach(() => {
+      port = new Port("name");
+      ship = jest.fn();
+      titanic = jest.fn();
+      queenMary = jest.fn();
+    });
+
+    it("can be instantiated", () => {
+      expect(new Port()).toBeInstanceOf(Object);
+    });
+
+    it("port has a name", () => {
+      expect(port.name).toBeDefined();
+    });
+
+    it("can add a ship", () => {
+      port.addShip(ship);
+
+      expect(port.ships).toContain(ship);
+    });
+
+    it("can remove a ship", () => {
+      port.addShip(titanic);
+      port.addShip(queenMary);
+      port.removeShip(queenMary);
+
+      expect(port.ships).toEqual([titanic]);
+    });
```

Comment:
In both of these tests you have 3 different ships defined. You could edit the first test to make use of one of the ships used in your second test, removing the need for the `ship` variable, i.e.
```
port.addShip(titanic);

expect(port.ships).toContain(titanic);
```